rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -123,6 +131,14 @@ void rocalTensorInfo::reset_tensor_roi_buffers() {
     }
 }
 
+void rocalTensorInfo::reallocate_tensor_sample_rate_buffers() {
+    _sample_rate = std::make_shared<std::vector<float>>(_batch_size);
+    _sample_rate->resize(_batch_size);
```

Comment:
We have already allocated the _sample_rate for batch size in the above line, Then why are we doing resize again?